.gitlab-ci.yml

```diff
@@ -16,12 +16,9 @@ before_script:
     - git fetch origin 'refs/heads/*:refs/heads/*'
     # Get last remaining tags, if any.
     - git fetch --tags origin
```

Comment:
I don't think this is necessary anymore. It was there for the check_commits which has been removed

Reply:
Removed.